src/platform/site-wide/user-nav/components/SearchMenu.jsx

```diff
@@ -18,6 +19,10 @@ const ENTER_KEY = 13;
 export class SearchMenu extends React.Component {
   constructor(props) {
     super(props);
+    this.debouncedSuggestions = debounce(
```

Comment:
Sorry last comment 😂 What do you think of `debouncedGetSuggestions` or `debouncedFetchSuggestions`? I think I would default to thinking that `debouncedSuggestions` is a GET but maybe the other 2 options are clearer? Not sure tbh, up to you 🙂

Reply:
sounds good to me, I will make the edit